formats/config/common.go

```diff
@@ -0,0 +1,92 @@
+// Package config implements decoding, encoding and
+// manipulation git config files.
+//
+// Reference: https://git-scm.com/docs/git-config
+package config
+
+// New creates a new config instance.
+func New() *Config {
+	return &Config{}
+}
+
+type Config struct {
+	Comment  *Comment
+	Sections Sections
+	Includes Includes
+}
+
+type Includes []*Include
+
+// A reference to a included configuration.
+type Include struct {
+	Path   string
+	Config *Config
+}
+
+type Comment string
```

Comment:
Yep. It's still missing comment handling code such as returning comment without comment symbols (;. #).